src/HN/Model/Mailman.hs

```diff
@@ -0,0 +1,137 @@
+-- | construct a feed from a mailman (pipermail) archive,
+-- e.g., https://mail.haskell.org/pipermail/libraries/
+-- see https://github.com/chrisdone/haskellnews/issues/12
+-- and https://github.com/haskell-infra/hl/issues/63#issuecomment-74420850
+
+{-# language OverloadedStrings #-}
+{-# language BangPatterns #-}
+
+module HN.Model.Mailman where
+
+import Network.HTTP.Conduit
+-- I tried with
```

Comment:
Could you open an issue with the traces / or further information on why it wouldn't work? 

We could maybe see about fixing it.

Then you can remove this comment as well.